stdlib/LinearAlgebra/src/givens.jl

```diff
@@ -48,10 +48,13 @@ Rotation{T}(R::Rotation) where {T} = Rotation{T}([Givens{T}(g) for g in R.rotati
 AbstractRotation{T}(G::Givens) where {T} = Givens{T}(G)
 AbstractRotation{T}(R::Rotation) where {T} = Rotation{T}(R)
 
-adjoint(G::Givens) = Adjoint(G)
+adjoint(G::Givens) = Givens(G.i1, G.i2, G.c', -G.s)
 adjoint(R::Rotation) = Adjoint(R)
-Base.copy(aG::Adjoint{<:Any,<:Givens}) = (G = aG.parent; Givens(G.i1, G.i2, conj(G.c), -G.s))
-Base.copy(aR::Adjoint{<:Any,Rotation{T}}) where {T} = Rotation{T}(reverse!([copy(r') for r in aR.parent.rotations]))
+function Base.copy(aG::Adjoint{<:Any,<:Givens})
+    G = aG.parent
+    return Givens(G.i1, G.i2, conj(G.c), -G.s)
+end
+Base.copy(aR::Adjoint{<:Any,Rotation{T}}) where {T} = Rotation{T}(reverse!([r' for r in aR.parent.rotations]))
```

Comment:
Couldn't you do non-mutating `reverse(aR.parent.rotations)` instead of mutating a comprehension?

Reply:
You still need to map the adjoint operation in any case.

Reply:
Oh derp, I'm not sure how I missed that.